Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1788,25 +1788,6 @@ def argToList(arg, separator=','):
     return [arg]
 
 
-def remove_duplicates_from_list_arg(args, field):
```

Comment:
is this intentional? I see it's in use, for example in `MicrosoftDefenderAdvancedThreatProtection.py`.